docs/2022-04-18-feature-toggles.md

```diff
@@ -10,18 +10,13 @@ This document describes the main features we are using and how to interact with
 
 ## Feature toggles
 
-We have a list of [feature toggles](https://docs.getunleash.io/advanced/feature_toggle_types) that are queried by the wallets:
+We have two [feature toggles](https://docs.getunleash.io/advanced/feature_toggle_types) that are queried by the wallets:
```

Comment:
Unrelated to PR.
In unleash we have a single project with all feature toggles, should we create a separate project for wallet-service and if we require more feature toggles we can add them there without having to update these docs?

Reply:
It would be great, but we're running the community version of Unleash and creating multiple projects are only available in the "Pro & Enterprise" version:

![image](https://github.com/HathorNetwork/hathor-wallet-service/assets/3586068/07fb3d47-d9e0-4b6a-b633-db92a323c58b)

But that's really valid point, as we're currently receiving feature toggles for all projects, it's not that bad atm because we have only a few feature toggles, but it's wasted bandwidth

I'll open an issue to study if we would benefit from upgrading and the pricing